docs/UserGuide.md

```diff
@@ -261,15 +261,15 @@ Finds clients whose names contain any of the given keywords.
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
 * The search is case-insensitive. e.g `hans` will match `Hans`.
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`.
```

Comment:
It would be good to not delete this as I will be modifying the `find by name` method to ensure search is also carried out in this manner.